common/utils/utils.go

```diff
@@ -225,28 +226,18 @@ func CreateGCSBucket(bucketName, projectID string) error {
 	}
 	defer client.Close()
 	bucket := client.Bucket(bucketName)
-	buckets := client.Buckets(ctx, projectID)
-	for {
-		if bucketName == "" {
-			return fmt.Errorf("bucketName entered is empty %v", bucketName)
-		}
-		attrs, err := buckets.Next()
-		// Assume bucket not found if at Iterator end and create
-		if err == iterator.Done {
-			// Create bucket
-			if err := bucket.Create(ctx, projectID, nil); err != nil {
+	if err := bucket.Create(ctx, projectID, nil); err != nil {
+		if e, ok := err.(*googleapi.Error); ok {
+			// Ignoring the bucket already exists error.
+			if e.Code != 409 {
```

Comment:
We should had a (WARN) log here letting the user now that the bucket already exists and we will re-use it.

Reply:
added a message to be printed for the user